schema/factory/CommonFactory.go

```diff
@@ -36,20 +40,60 @@ type CommonFactory struct {
 	zLogger     zerolog.Logger
 }
 
+type ZerologConfig struct {
+	Level      string `properties:"logger.th2.global_level,default=info"`
```

Comment:
Please remove the `logger.th2` prefix because we set global configure for all zero logger in the whole app in our code and other code are included into the app